suse_migration_services/migration_config.py

```diff
@@ -53,13 +54,20 @@ def get_migration_product(self):
         The value returned is passed to the --product option in the
         zypper migration call
         """
-        migration_product = self.config_data.get('migration_product')
-        if not migration_product:
+        migration_product = None
+        suse_base_product = SUSEBaseProduct()
+        if suse_base_product.is_version_supported():
+            name = suse_base_product.get_tag('name')[0]
+            arch = suse_base_product.get_tag('arch')[0]
+            migration_product = '/'.join(
+                [name, str(suse_base_product.get_default_target_version()), arch]
+            )
```

Comment:
The creation of the product string for the later use with the zypper migration plugin is a responsibility of the SUSEBaseProduct class. I think the class should provide a method that just returns this information, e.g

```python
migration_product_name = suse_base_product.get_product_name()
```

Reply:
The ```get_product_name``` method implementation imho should contain the code that reads the tags and constructs the name